content/applications/hr/payroll/reporting.rst

```diff
@@ -2,218 +2,257 @@
-be downloaded into an Excel spreadsheet.
+To export the data in an XLSX format, click the :guilabel:`Download xlsx` icon. The information is
+then downloaded into a spreadsheet.
 
-.. image:: reporting/export.png
+.. image:: reporting/xlsx.png
    :align: center
-   :alt: Download an Excel spreadsheet of the data by clicking the download button.
+   :alt: The menu options with the download button highlighted.
 
-The data can be inserted into a spreadsheet by clicking the :guilabel:`Insert in Spreadsheet`
-button. A pop-up appears asking which spreadsheet to place the information in. Select an existing
-spreadsheet or enter the name for a new spreadsheet. Click the :guilabel:`Confirm` button to move to
-a spreadsheet view with all the information populated.
+Any report can be inserted into a spreadsheet by clicking the :guilabel:`Insert in Spreadsheet`
+button. A :guilabel:`Select a spreadsheet to insert your (type of report)` pop-up window appears,
+asking which spreadsheet to place the information in. Select an existing spreadsheet or dashboard,
+or select a new :guilabel:`Blank spreadsheet`. Click the :guilabel:`Confirm` button to move to a
+spreadsheet view with the report added to it.
 
 .. image:: reporting/spreadsheet.png
    :align: center
-   :alt: Send the data to a spreadsheet.
-
-Once in the spreadsheet view, click on :guilabel:`File`, then select :guilabel:`Save` to save the
-data, or :guilabel:`Save as Template` to save the setting as a template to use in the future.
+   :alt: The view of data sent to a spreadsheet.
 
 .. note::
-   The *Documents* application must be installed in order to use the :guilabel:`Insert in
-   Spreadsheet` option.
+   If the *Documents* app is **not** installed, the :guilabel:`Insert in Spreadsheet` option places
+   the newly created spreadsheet in the *Dashboards* app.
 
+   If the *Documents* application **is** installed, the spreadsheet has the option to be stored in
+   either the *Dashboards* app or *Documents* app.
 
-Meal Vouchers
--------------
+Work entry analysis
+-------------------
 
-:guilabel:`Meal Vouchers` provides an overview of the meal vouchers used by employees, and can be
-shown by :guilabel:`Day`, :guilabel:`Week`, :guilabel:`Month`, :guilabel:`Quarter`, or
-:guilabel:`Year`. The default view is by month.
+The default :guilabel:`Work entry analysis` report provides an overview of the validated work
+entries for the current month. To view this report, navigate to :menuselection:`Payroll app -->
+Reporting -> Work Entry Analysis`.
 
-.. image:: reporting/meal-vouchers.png
-   :align: center
-   :alt: Meal voucher
+To change the displayed view, click the :guilabel:`➕ (plus)` icon next to the word
+:guilabel:`Total`, then click on one of the grouping options. The available options are
+:guilabel:`Work Entry Type`, :guilabel:`Employee`, and :guilabel:`Department`. If in a multi-company
+database, a :guilabel:`Company` option also appears.
```

Comment:
this portion should be reworked to cleanly introduce the default view and filter on this reporting page -- at the moment I don't think it's as "detailed" as it could be.

Reply:
Good idea- it was so minimal, I almost thought it was obvious (since it just shows the totals of what the name of the report is) but that's a great suggestion- added!